auth4genai/intro/account-linking.mdx

```diff
@@ -7,50 +7,29 @@ description: "Account linking in Auth0 allows a user to connect multiple identit
 
 For AI agents that need to act on a user's behalf, account linking is critical. An agent might need to access a user's calendar via their Google account and their code repositories through their GitHub account. Without account linking, the agent would see these as two separate users.
 
-By linking these accounts, your AI agents have a holistic view of the user. This unified profile is essential for Token Vault to retrieve the correct tokens for the various external services the agent needs to call. When an agent requests to access a third-party API, account linking ensures that Auth0 can associate that request with the current user and their authorized connections.
+By linking these accounts, your AI agents have a holistic view of the user. This unified profile is essential for Token Vault to retrieve the correct tokens for the various external services the agent needs to call. When an agent requests to access an external API, account linking ensures that Auth0 can associate that request with the current user and their authorized connections.
 
-### How it works
-
-The account linking process merges two user profiles into a primary and a secondary account. The user_id and core profile properties of the primary account are retained, and the secondary account's identity is added to the primary profile's identities array. Multiple secondary accounts can be added to a primary account by linking.
-
-There are two primary methods for implementing account linking:
+[Connected Accounts for Token Vault](https://auth0.com/docs/secure/tokens/token-vault/connected-accounts-for-token-vault) enables a unified Auth0 user profile to be linked to multiple external accounts, such as Google, GitHub, or Slack, where users can log in to any one of their accounts using a single set of credentials.
```

Comment:
"where users can log in to any one of their accounts using a single set of credentials."

this is not accurate. Each connected accounts still have their own credentials, but after the user has connected their external account once, the application is able to call external APIs on behalf of the user without re-authenticating anymore (well, until the refresh token to the provider expires).